scripts/tools/metricTracer.yml

```diff
@@ -0,0 +1,261 @@
+    - type: $t_value_t 
+      name: "value"
+      desc: "[out] value of the decodable metric entry or event. Number is meaningful based on the metric type."
+    - type: uint64_t  
+      name: "timeStamp"
+      desc: "[out] timestamp at which the event happened."
+    - type: uint32_t
+      name: "metricIndex"
+      desc: "[out] index to the decodable metric handle in the input array (phMetric) in $tMetricTracerDecode()"
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Create a metric tracer for a device."
+version: "1.10"
+class: $tMetricTracer
+name: CreateExp
+decl: static
+details:
+    - "The notification event must have been created from an event pool that was created using $X_EVENT_POOL_FLAG_HOST_VISIBLE flag."
+    - "The duration of the signal event created from an event pool that was created using $X_EVENT_POOL_FLAG_KERNEL_TIMESTAMP flag is undefined. However, for consistency and orthogonality the event will report correctly as signaled when used by other event API functionality."
+    - "The application must **not** call this function from simultaneous threads with the same device handle."
+    - "The metric tracer is created in disabled state"
+    - "Metric groups must support sampling type ZET_METRIC_SAMPLING_TYPE_EXP_FLAG_TRACER_BASED"
+    - "All metric groups must be first activated"
+params:
+    - type: $t_context_handle_t
+      name: hContext
+      desc: "[in] handle of the context object"
+    - type: "$t_device_handle_t"
+      name: hDevice
+      desc: "[in] handle of the device"
+    - type: uint32_t
+      name: "metricGroupCount"
+      desc: "[in] metric group count"
+    - type: "$t_metric_group_handle_t*"
+      name: "phMetricGroups"
+      desc: "[in][range(0, metricGroupCount )] handles of the metric groups to trace"
+    - type: "$t_metric_tracer_exp_desc_t*"
+      name: desc
+      desc: "[in,out] metric tracer descriptor"
+    - type: "$x_event_handle_t"
+      name: hNotificationEvent
+      desc: "[in][optional] event used for report availability notification. Note: If buffer is not drained when the event it flagged, there is a risk of HW event buffer being overrun"
+    - type: "$t_metric_tracer_exp_handle_t*"
+      name: phMetricTracer
+      desc: "[out] handle of the metric tracer"
+returns:
+    - $X_RESULT_ERROR_INVALID_SYNCHRONIZATION_OBJECT
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Destroy a metric tracer."
+version: "1.10"
+class: $tMetricTracer
+name: DestroyExp
+details:
+    - "The application must **not** call this function from simultaneous threads with the same metric tracer handle."
+params:
+    - type: "$t_metric_tracer_exp_handle_t"
+      name: hMetricTracer
+      desc: "[in] handle of the metric tracer"
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Lightweight call that starts the event collections"
```

Comment:
what does "lightweight call" mean? also- it would be good to move description such as this to "details" section and not in the description of the API itself

Reply:
desc:  "Start events collection"
detail:  -"Driver implementations must make this API call have as minimal overhead as possible, to allow applications start/stop event collection at any point during execution"

Acceptable? 

Also appliable to disable below